### PR TITLE
Add MIT license

### PR DESCRIPTION
**Pull request by bradvogel**
closes https://github.com/pelias/parser/issues/73

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Pelias, Inc
```
Comment (Joxit) [inline LICENSE.md]: Should be `Pelias Contributors` and not `Pelias, Inc` ?

**Comment by bradvogel** [comment]
fixed

**Comment by missinglink** [comment]
LGTM, @Joxit you requested changes, so I'll leave it for you to merge :)

**Comment by Joxit** [comment]
Thank you @bradvogel for your contribution :)

**Pull request merged**